gr-uhd/lib/usrp_sink_impl.cc

```diff
@@ -388,7 +388,7 @@ int usrp_sink_impl::work(int noutput_items,
             // There is a tag gap since no length_tag was found immediately following
             // the last sample of the previous burst. Drop samples until the next
             // length_tag is found. Notify the user of the tag gap.
-            std::cerr << "tG" << std::flush;
+            GR_LOG_ERROR(d_logger, "tG");
```

Comment:
wait, does this mean that this stuff will come one per line from now on? Should it? Should OA and uA, O, etc too?

Reply:
That is an excellent question, and that's why I'm hoping for feedback from his UHDness, the @mbr0wn himself

Reply:
> wait, does this mean that this stuff will come one per line from now on? Should it? Should OA and uA, O, etc too?

UOSDL come from UHD, not GNU Radio. Unless you want to pipe UHD's logging through GNU Radio, this isn't really something we can decide here.

Note that even UHD treats UOSDL specially: Because they come from the fast path, and by design should *not* be one-per-line, they go through a different logging path.

Reply:
Note that I'm crazy enough to say "UHD has a good logging system, we can basically hook up anything we want to it, including the GR logging system", and *aim* for that functionality (with a highly uncertain time frame).

The question I'm asking myself here is whether tG in GR schould be handled like UOSDL in UHD, or like any other log message.